examples/css/style.css

```diff
@@ -31,4 +31,10 @@ canvas.example-canvas {
 .column pre {
     width: 100%;
     overflow-x: auto;
+}
+
+#example7-canvas {
+    background-image: url(/examples/assets/hacktoberfest2.webp);
```

Comment:
```suggestion
    background-image: url(../assets/hacktoberfest2.webp);
```

Relative URL would be better.

Reply:
Because with absolute url it doesn't work if the example is in subfolder. If you change that, I will accept the PR. @Rasmuskrogh

Reply:
I've fixed it internally on my machine. I'll send a new pull request.

Reply:
You can just click here :)

Reply:
I think i messed up by pushing it again. Now my changes are outdated and the commit suggestion is disabled :/

Reply:
Ah, ok, that is also good! Thank you!